www/js/app.js

```diff
@@ -218,19 +220,79 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
         return true;
     });
     // Top menu :
+    function addAnimation(section) {
+            if(section == "home") { 
+                if(!$("#configuration").is(":hidden")) {
+                    $('#configuration').addClass("slideOut_R");
+                    setTimeout(function() {
+                        $('#configuration').hide();
+                    }, 300); 
+                }
+                if(!$("#about").is(":hidden")) {
+                    $('#about').addClass("slideOut_R");
+                    setTimeout(function() {
+                        $('#about').hide();
+                    }, 300); 
+                }
+                $('#articleContent').addClass("slideIn_R");
+                setTimeout(function() {
+                    $('#articleContent').show();
+                }, 300);
+            } else if(section == "config") {
+                if(!$("#about").is(":hidden")) {
+                    $('#about').addClass("slideOut_R");
+                    $('#configuration').addClass("slideIn_R");
+                    setTimeout(function() {
+                        $('#about').hide();
+                    }, 300); 
+                } else if(!$("#articleContent").is(":hidden")) {
+                    $('#articleContent').addClass("slideOut_L");
+                    $('#configuration').addClass("slideIn_L");
+                    setTimeout(function() {
+                        $('#articleContent').hide();
+                    }, 300);
+                }
+                setTimeout(function() {
+                    $('#configuration').show();
+                }, 300); 
+            } else if(section == "about") {
+                if(!$("#configuration").is(":hidden")) {
+                    $('#configuration').addClass("slideOut_L");
+                    setTimeout(function() {
+                        $('#configuration').hide();
+                    }, 300); 
+                }
+                if(!$("#articleContent").is(":hidden")) {
+                    $('#articleContent').addClass("slideOut_L");
+                    setTimeout(function() {
+                        $('#articleContent').hide();
+                    }, 300); 
+                }
+                $('#about').addClass("slideIn_L");
+                setTimeout(function() {
+                    $('#about').show();
+                }, 300);     
+            }
+        
+    }
     $('#btnHome').on('click', function(e) {
         // Highlight the selected section in the navbar
         $('#liHomeNav').attr("class","active");
         $('#liConfigureNav').attr("class","");
         $('#liAboutNav').attr("class","");
         $('.navbar-collapse').collapse('hide');
         // Show the selected content in the page
-        $('#about').hide();
-        $('#configuration').hide();
+        uiUtil.removeAnimationClasses();
+        if(params.showUIAnimations) { 
```

Comment:
Missing space after `if` (to be consistent with our house style)

Reply:
Ok... I will make sure this in my next commit